scripts/sign_dlls.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python3
+"""
+Script to harvest and restore dlls for signing purposes.
+"""
+
+__author__ = "Maarten Tegelaers"
```

Comment:
I believe this is a copy of an existing script developed by maarten, probably part of another project. i have a question, but i think it is outside the scope of this task

Reply:
Not exactly. This modified to:
- look for dlls generated by both csprojs and vcxprojs
- look in PROJECT_BINARY_DIR as in `/path/to/build/proj/Release` subdirs rather `bin`
So it behaves differently compared to 
https://github.com/Deltares/D-HYDRO-Automation-Scripts/blob/main/gather_to_sign_dll/gather_to_sign_dll.py
Therefore it is part if the current project, and not external dependency in TeamCity.